openstates/in/bills.py

```diff
@@ -77,9 +81,127 @@ def get_name(self,random_json):
         #got sick of doing this everywhere
         return random_json["firstName"] + " " + random_json["lastName"]
 
+    def process_votes(self,rollcalls,bill,proxy):
+
+        result_types = {
+            'FAILED': False,
+            'DEFEATED': False,
+            'PREVAILED': True,
+            'PASSED': True,
+            'SUSTAINED': True,
+            'NOT SECONDED': False,
+            'OVERRIDDEN': True,
+            'ADOPTED': True,
+        }
+
+        for r in rollcalls:
+            proxy_link = proxy["url"] + r["link"]
+            (path, resp) = self.urlretrieve(proxy_link)
+            text = convert_pdf(path, 'text')
+            lines = text.split("\n")
+            os.remove(path)
+
+
+            chamber = "lower" if "house of representatives" in lines[0].lower() else "upper"
+            date_parts = lines[1].strip().split()[-3:]
+            date_str = " ".join(date_parts).title() + " " + lines[2].strip()
+            vote_date = datetime.datetime.strptime(date_str,"%b %d, %Y %I:%M:%S %p")
+
+            for res,val in result_types.items():
+                if res in lines[3].upper():
+                    passed = val
+                    break
+
+            motion = " ".join(lines[4].split()[:-2])
+            try:
+                yeas = int(lines[4].split()[-1])
+                nays = int(lines[5].split()[-1])
+                excused = int(lines[6].split()[-1])
+                not_voting = int(lines[7].split()[-1])
+            except ValueError:
+                self.logger.warning("Vote format is weird, skipping")
+                continue
```

Comment:
How often does this come up? Was it just that single weird bill you had talked about with the me the other day?